Deceive/ConfigProxy.cs

```diff
@@ -38,10 +38,12 @@ public static int StartConfigProxy(string configUrl, int chatPort)
 
             // Run this on a new thread, just for the sake of it.
             // It seemed to be buggy if run on the same thread.
-            new Thread(() =>
+            var thread = new Thread(() =>
             {
                 server.RunAsync().Wait();
-            }).Start();
+            });
+            thread.IsBackground = true;
```

Comment:
This exits the thread, when Deceive wants to exit. Otherwise Deceive can only be forced quit.